.travis.yml

```diff
@@ -60,6 +60,8 @@ install:
 - |
   if [[ "$RUNTYPE" == "release" ]]; then
     sudo snap install snapcraft --classic
+  else
+    make protoc
```

Comment:
This should be a part of the `RUNTYPE` script, where this is necessary(`js.test` I suppose?), same as it's done for all other `RUNTYPE`s

Reply:
It doesn't belong in `script` either. We're cleaning up this mess in #11 and #377 and I really don't feel like doing it again in this PR, so let's please just merge this and get #377 moving forward.